Automatic Validation ended with:
> 2024-04-23T11:10:27.7031531Z ##[error]Manifest Validation Failed
 2024-04-23T11:10:32.7563377Z ##[section]Finishing: Validate Manifest

(Automated response - build 877.)

--- a/manifests/b/BellSoft/LibericaJDK/11/11.0.23.10/BellSoft.LibericaJDK.11.installer.yaml
+++ b/manifests/b/BellSoft/LibericaJDK/11/11.0.23.10/BellSoft.LibericaJDK.11.installer.yaml
@@ -12,11 +12,11 @@ Installers:
   AppsAndFeaturesEntries:
   - DisplayName: Liberica JDK 11 (64-bit)
 - Architecture: x86
-InstallerUrl: https://download.bell-sw.com/java/11.0.23+10/bellsoft-jdk11.0.23+10-windows-i586.msi
-InstallerSha256: 60EBCFC2E5C91C8C6EBE7C5433A06F071DA45569B3A23426D654D2BB35989718
-ProductCode: '{1AAB2CEE-3437-431F-B89F-66307C522931}'
-AppsAndFeaturesEntries:
-- DisplayName: Liberica JDK 11 (32-bit)
+  InstallerUrl: https://download.bell-sw.com/java/11.0.23+10/bellsoft-jdk11.0.23+10-windows-i586.msi
+  InstallerSha256: 60EBCFC2E5C91C8C6EBE7C5433A06F071DA45569B3A23426D654D2BB35989718
+  ProductCode: '{1AAB2CEE-3437-431F-B89F-66307C522931}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Liberica JDK 11 (32-bit)
 - Architecture: arm64
   InstallerUrl: https://download.bell-sw.com/java/11.0.23+10/bellsoft-jdk11.0.23+10-windows-aarch64.msi
   InstallerSha256: E292990F8BD6292FFBB73B11CA398C226E354381E95BFFC75D0B00F35169698C

